framework/doc/content/source/preconditioners/HMGPreconditioner.md

```diff
@@ -0,0 +1,65 @@
+# HMG (High-performance (Hybrid) multigrid method)
+
+## Overview
+
+[HMG](https://www.mcs.anl.gov/petsc/petsc-current/docs/manualpages/PC/PCHMG.html) 's
```

Comment:
Space between `)` and `'`. Hopefully rendering works fine if you remove that space?